opendevin/server/session/session.py

```diff
@@ -15,13 +23,22 @@ class Session:
     websocket: WebSocket | None
     last_active_ts: int = 0
     is_alive: bool = True
+    agent: AgentSession
```

Comment:
```suggestion
    agent_session: AgentSession
```
It was renamed everywhere but here? If I'm seeing this right

Reply:
Whoops good catch!